docs/using/installing/index.rst

```diff
@@ -677,6 +677,6 @@ So ensure that the firewall allows access to TCP port 22 on each node from the e
 Next Steps
 ----------
 
-Next, we will describe how to use cluster security and authentication.
+The next section describes how to set up an :ref:`authenticated user<authenticate>`.
 However, you may want to perform the steps in :ref:`the MongoDB tutorial <movingapps>` to ensure that your nodes are correctly configured.
```

Comment:
This needs to be removed and placed at the bottom of the next page. It is impossible to proceed with the MongoDB tutorial without creating an authenticated user.

Put another way, at this point creating a user is the ONLY thing a user can do, and they MUST do it.